packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -152,9 +152,11 @@ async function main() {
         'build+test+package': 'npm run build+test && npm run package',
         'build+test': 'npm run build && npm test',
         compat: 'cdk-compat',
+        gen: 'cfn2ts',
       },
       'cdk-build': {
         cloudformation: namespace,
+        pre: 'npm run gen',
```

Comment:
It is supposed to be a list of strings. But better yet, don't do this at all.

Reply:
Ouch, thanks!